packages/infra/config/env-config.ts

```diff
@@ -84,6 +84,7 @@ export type EnvConfig = {
       SIDECHAIN_FHIR_CONVERTER_KEYS: string;
     };
   };
+  fhirToCDAUrl?: string;
```

Comment:
Is this part of the scope of this PR, or a fix you're shipping as well? If a fix, its a good idea to add a comment to your own code when its something that reviewers are not expecting to see. :)

Reply:
Good point! Yes its a fix that I added since that variable being undeclared was causing errors for me when running code. Noted for next time!

Reply:
Wait, this is odd: this property is on `develop` - [link](https://github.com/metriport/metriport/blob/develop/packages/infra/config/env-config.ts#L87)

One would think that your branch is way behind `develop`, but then it would compare with `develop` and we'll see only your PR trying to make it optional while its required on `develop`.

But your branch is newer than that change - ~2 weeks.

Nonetheless, maybe rebase your branch w/ `develop`?